src/screens/AdminPortal/ManageTag/editModal/EditUserTagModal.spec.tsx

```
@@ -41,107 +30,147 @@ describe('EditUserTagModal Component', () => {
-    fireEvent.click(screen.getByTestId('closeEditTagModalBtn'));
-    expect(defaultProps.hideEditUserTagModal).toHaveBeenCalledTimes(1);
-  });
-
-  it('calls handleEditUserTag when form is submitted with valid input', async () => {
-    render(<EditUserTagModal {...defaultProps} />);
-
-    fireEvent.click(screen.getByTestId('editTagSubmitBtn'));
-
+  it('should submit form when valid', async () => {
+    render(<EditUserTagModal {...defaultProps} newTagName="Valid Name" />);
+    
```

ESLint

[error] Delete `····`

(prettier/prettier)